<launch>
    <node pkg="tf2_ros" type="static_transform_publisher" name="word_slam" args="0 0 0  0.5, -0.5, 0.5, -0.5  world slam" />

    <node pkg="disinfslam" type="simulate_disinf_slam" name="simulate_disinf_slam" output="screen" >
        <param name="model_path" value="/home/yu/tools/ht_lt.pt" />
        <param name="calib_path " value="$(find disinfslam)/configs/simulation.yaml" />
        <param name="orb_vocab_path" value="/home/yu/Downloads/orb_vocab/orb_vocab.dbow2" />
        <param name="renderer" type="bool" value="false" />
        <param name="use_mask" type="bool" value="false" />
        <param name="bbox_xy" type="double" value="8.0" />
        <remap from="/stereoLeft" to="/zed2/zed_node/left/image_rect_color" />
        <remap from="/stereoRight" to="/zed2/zed_node/right/image_rect_color" />
        <remap from="/depth" to="/camera/aligned_depth_to_color/image_raw" />
        <remap from="/rgbImg" to="/camera/color/image_raw" />

    </node>
</launch>